### ISSUE TITLE
PR #95 causing error dialog when unfixable ESLint error is encountered.

### ISSUE THREAD

**Issue by arjunmehta** [elicwhite/ESLint-Formatter#97]
@elicwhite @codesculpture 

I think PR #95 is causing a major issue where now every ESLint Error, when there is an unfixable error, it is throwing an error.

Normally this would not throw an error in the same way and would just leave the unfixable error alone.

I am using the project's ESLint bin, not sure if that has to do with it.

In SublimeText Console:
```
Using node.js path on 'osx': /Users/myuser/.nvm/versions/node/v20.18.1/bin/node
Using eslint path on 'osx': /Users/myuser/myproject/node_modules/.bin/eslint
Unexpected error(<class 'Exception'>): Error: 
error: Error: 
```

Error dialog:
<img width="404" alt="Image" src="https://github.com/user-attachments/assets/dc23d62f-9aa3-4a2d-884c-7b33986553e0" />
